src/event_mapper.py

```diff
@@ -4,19 +4,23 @@
 
 EVENT_ID = 'eventId'
 EVENT_TYPE = 'eventType'
-TIMESTAMP = 'timestamp'
+TIME_STAMP = 'time_stamp'
```

Comment:
Can we stick with `snake_case` or `camelCase` for these variables please?

Reply:
Presumably this is a behaviour change too as the hub will be sending events with `timestamp` as a field?

Reply:
I used time_stamp for consistency because timestamp is a keyword in PostgreSQL. However, hub components are using timestamp. I will change Python code to use timestamp as a variable and PostgreSQL will continue to use time_stamp.

Reply:
Done.